scss/components/_common.scss

```diff
@@ -0,0 +1,40 @@
+body {
```

Comment:
you cant have 'common styles' -- you can only have blocks -- one scss file for one block